packages/web/src/js/Collapse.ts

```diff
@@ -27,6 +27,10 @@ interface CollapseState {
   width: number;
 }
 
+const elementHasAriaExpanded = (node: HTMLElement) => {
```

Comment:
Why is this not a class method?

You should also use explicit return of arrow function.